thirdeye/thirdeye-frontend/app/helpers/compute-color.js

```diff
@@ -11,9 +11,8 @@ export function computeColor([value = 0]) {
 
   if (value > 0) {
     return `rgba(0,0,234,${opacity})`;
-  } else{
-    return `rgba(234,0,0,${opacity})`;
   }
+  return `rgba(234,0,0,${opacity})`;
```

Comment:
would it be cleaner to have a ternary ?

i.e.    return value>0 ? `rgba(0,0,234,${opacity})` : `rgba(234,0,0,${opacity})`;

Reply:
Yes, it could be. I haven't changed it because my purpose is fixing eslint error in the codebase and changing the code structure will involve more risk. I need to test it again. For this suggestion, I can change it because the logic is simple. Thanks